<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="ImWeb_file_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Open</set>
   </property>
  </widget>
  <widget class="QLabel" name="ImWeb_file">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>35</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Naver_file">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>75</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="Naver_file_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Open</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Format_func_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>140</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>변환</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="Save_to_btn">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>210</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Save_path">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>249</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Save_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
